### ISSUE TITLE
Incomplete Speech Output for Metronome toggle with screen reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30480]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1.  With NVDA screen reader enabled open a score and press Ctrl M a few times to toggle the metronome on and off.

Notice that each time you do this the screen reader just announces "Metronome" but does not say whether the metronome has been toggled on or off.  Ideally this would say "Metronome On" and "Metronome Off".

In 4.5 this key press did not result in any speech.  Speech should announce a change of state or an action that has happened rather than simply echo a key press function.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.1

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
